### ISSUE TITLE
CodeSnippets can't display if `languageTabs` isn't present in docusaurus theme config

### ISSUE THREAD

**Issue by ElliotFriend** [PaloAltoNetworks/docusaurus-openapi-docs#870]
Hi, @sserrata!! 👋🏻 

Super excited for the `v3.0.0` release! I know it's been a lot of work, and I'm so thankful for everything you've done!!

Working on getting my project updated, and I hit a snag with the recent code snippets ordering feature. Looks like if `siteConfig.themeConfig.languageTabs` is either a) absent from the config, or b) an empty array, docusaurus will hit a runtime error complaining: `TypeError: Cannot read properties of undefined (reading 'samples')`

![Screenshot 2024-07-03 at 2 55 19 PM](https://github.com/PaloAltoNetworks/docusaurus-openapi-docs/assets/2024293/37482456-889f-4c00-b79f-a66c0008eecb)

I think it might be crashing because if `userDefinedLanguageSet` is nullish, the filtered and then merged language set ends up being empty.

I was able to use [patch-package](https://github.com/ds300/patch-package) to patch the built plugin. Here is the diff that solved my problem:

```diff
diff --git a/node_modules/docusaurus-theme-openapi-docs/lib/theme/ApiExplorer/CodeSnippets/index.js b/node_modules/docusaurus-theme-openapi-docs/lib/theme/ApiExplorer/CodeSnippets/index.js
index 53f4f65..e5ab3ca 100644
--- a/node_modules/docusaurus-theme-openapi-docs/lib/theme/ApiExplorer/CodeSnippets/index.js
+++ b/node_modules/docusaurus-theme-openapi-docs/lib/theme/ApiExplorer/CodeSnippets/index.js
@@ -220,7 +220,7 @@ function CodeSnippets({ postman, codeSamples }) {
   const auth = (0, hooks_1.useTypedSelector)((state) => state.auth);
   // User-defined languages array
   // Can override languageSet, change order of langs, override options and variants
-  const userDefinedLanguageSet = siteConfig?.themeConfig?.languageTabs;
+  const userDefinedLanguageSet = siteConfig?.themeConfig?.languageTabs || exports.languageSet;
   // Filter languageSet by user-defined langs
   const filteredLanguageSet = exports.languageSet.filter((ls) => {
     return userDefinedLanguageSet?.some((lang) => {
```

_This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296)._

**Comment by sserrata** [comment]
Hi @ElliotFriend, I was just troubleshooting this very issue. I'll try to get a fix out ASAP. Thanks!

**Comment by sserrata** [comment]
Released v3.0.1 with fix
